src/Elastic.Markdown/Slices/_Layout.cshtml

```diff
@@ -5,28 +5,35 @@
 	<title>@Model.Title</title>
 	<link rel="stylesheet" type="text/css" href="@Model.Static("styles.css")"/>
 	<meta charset="utf-8">
-	@* <meta name="viewport" content="width=device-width, initial-scale=1.0"> *@
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta name="robots" content="@(Model.AllowIndexing ? "index, follow" : "noindex, nofollow")">
+	<meta name="pages-nav-current-page-id" content="@Model.CurrentDocument.Id">
+	<meta name="pages-nav-current-page-parent-ids" content="@(string.Join(",", Model.ParentIds))">
+	<meta name="pages-nav-current-navigation" content="@LayoutViewModel.CurrentNavigationId">
```

Comment:
This data is actually not needed anymore because we don't need to recreate the navigation state. But kept it here just in case.